### PR TITLE
add climate change image and hackbar

### PR DESCRIPTION
**Pull request by fernanda-costa**

### CODE REVIEW

File: _hackbars/fernanda_costa.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: hackbar
+title: Open Source
+author: Fernanda Costa
+---
+
+Getting started to contribute to open source. Thanks for providing this opportunity.
+---
+
+Humans are polar bears only predator. Stop global warming. Save the polar bears.
```
Comment (tasnimzotder) [inline _hackbars/fernanda_costa.md]: hey @fernanda-costa Can you change the line - 
```suggestion
## Humans are polar bears' only predator. Stop global warming. Save the polar bears.
```

Comment (fernanda-costa) [reply _hackbars/fernanda_costa.md]: Sure :) sry

Comment (tasnimzotder) [reply _hackbars/fernanda_costa.md]: it's okay

**Review by tasnimzotder — APPROVED** [review]

**Comment by tasnimzotder** [comment]
Thanks @fernanda-costa, for your contributions :)

**Pull request merged**